src/%IPM/Repo/Filesystem/Definition.cls

```diff
@@ -259,7 +260,7 @@ Method GetModuleStreamFromFile(pFilename As %String, Output pStream As %Stream.G
 		$$$ThrowOnError(tSC)
 		
 		Set tMetaStream.LineTerminator = $Select($$$isWINDOWS:$c(13,10),$$$isUNIX:$c(10))
-		Set pName = $ZConvert(tMetaStream.ReadLine(), "L")
+		Set pName = tMetaStream.ReadLine()
```

Comment:
why?

Reply:
Let's hold on this as well for the uppercase-lowercase package name discussion! Thanks